ingestion/src/metadata/ingestion/source/database/dbt/dbt_utils.py

```diff
@@ -44,6 +44,20 @@ def create_test_case_parameter_definitions(dbt_test):
                 }
             ]
             return test_case_param_definition
+        if hasattr(dbt_test, "freshness"):
+            test_case_param_definition = [
+                {
+                    "name": "warn_after",
+                    "displayName": "warn_after",
+                    "required": False,
+                },
+                {
+                    "name": "error_after",
+                    "displayName": "error_after",
+                    "required": False,
+                },
+            ]
+            return test_case_param_definition
```

Comment:
Can this be gotten from the dbt_test object similar to how we do it above?

Reply:
it cannot, freshness check has different model than regular test: https://schemas.getdbt.com/dbt/sources/v3/index.html#tab-pane_results_items_anyOf_i1